Components/CommentCard.jsx

```diff
@@ -1,14 +1,49 @@
 import Stack from 'react-bootstrap/Stack';
+import UserContext from "./UserContext"
+import { useContext, useState } from "react"
+import { deleteComment } from "./api";
+import Error from './Error';
+
+
+export default function CommentCard({comment, index, setCommentCount}){
+    const {loggedInUser} = useContext(UserContext)
+    const { username } = loggedInUser;
+    const [error, setError] = useState(null)
+    const [isLoading, setIsLoading] = useState(true)
+
+    const handleClick = (e) => {
+            e.preventDefault()
+            const commentid = comment.comment_id
+            deleteComment(commentid)
+            .then(() =>{
+                setCommentCount((commentCount) => commentCount - 1)
+                setIsLoading(false)
+            }).catch((err) => {
+                setCommentCount((commentCount) => commentCount + 1)
+                setError({
+                    status: 408,
+                    statusText: 'Unable to delete comment, please retry'})
+                setIsLoading(false)
```

Comment:
Well done considering errors, but why would a `408` be required here? We don't need to consider status codes on the front end in this way.